metrics/utils/version.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2022 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package metricutils
+
+import (
+	"github.com/klaytn/klaytn/params"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+func init() {
+	g := prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   metricNamespace,
+		Subsystem:   "",
+		Name:        "build_info",
+		Help:        "A metric with a constant '1' value labeled by version",
+		ConstLabels: prometheus.Labels{"version": params.Version},
```

Comment:
`ConstLabels` is commonly used to include some metadata like version, build info and so on.
https://pkg.go.dev/github.com/prometheus/client_golang/prometheus#Opts